apps/vaccitron/src/environments/environment.ts

```diff
@@ -7,5 +7,6 @@ export const environment = {
   vaccinesDataUrl: '../../assets/demo-data.json',
```

Comment:
'http://localhost:7071/api/Reports'
(If you prefer to have the FE debuggable without BE by default, please leave another line with this value in a comment, so it's easy to switch).